### DISCUSSION TITLE
Deta Space Builder item deletion

### DISCUSSION THREAD

**Discussion by silvioprog** [deta/cloud-docs#595] [Help] [answered]
Hey team.

Please take a look at this builder below and notice item `nextjs-demo`:

<img width="995" alt="Screen Shot 2022-11-15 at 11 00 51 PM" src="https://user-images.githubusercontent.com/1456829/202065465-de40592b-731c-4728-a986-5da76118d107.png">

It was created just for testing purposes to understand Deta Space, now I'm trying to delete this and clean my Space to create real projects.

So, how to delete this? 🤔

TIA for any help!

**Comment by lemonyte** [comment, marked as answer]
There is currently no way to delete Builder projects, though hopefully this feature comes soon. Don't worry about it taking up space, you can create as many projects as you like.

**Comment by silvioprog** [comment]
The deletion will be really a very useful feature. 👏 
Thanks a lot @LemonPi314!
